ps_checkout.php

```diff
@@ -1018,6 +1018,7 @@ public function hookActionFrontControllerSetMedia()
 
         Media::addJsDef([
             $this->name . 'Version' => $version->getSemVersion(),
+            $this->name . 'Locale' => $this->context->language->iso_code,
```

Comment:
`$this->context->language->iso_code` returns only the language code (e.g., `en`, `fr`), not a full locale code.

The variable name `locale` is currently misnamed as it actually holds a country language code. To improve clarity and accuracy, we should either rename the variable to reflect its current value or change the value format to follow standard locale codes.
 
To comply with standard locale formats and [PayPal requirements](https://developer.paypal.com/sdk/js/configuration/#link-locale), we should set a real locale code.

This can be done by constructing the locale code using both the language and country codes, ensuring it is reusable and compliant with PayPal's standards.

Reply:
While I would agree with you, this variable is only used for frontoffice-sdk translations and not in PayPal API